code/game/objects/items/weapons/clown_items.dm

```diff
@@ -165,6 +165,26 @@
 					continue
 			M.emote("flip")
 
+/obj/item/weapon/bikehorn/saxophone

```

Comment:
Oh in that case you fucked up something, because this should function **identical** to the bikehorn, other than the Crossed() proc just having `return` in it